nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/UpdateDocument.java

```diff
@@ -53,19 +53,17 @@
     protected String changeToken = null;
 
     @OperationMethod(collector = DocumentModelCollector.class)
-    public DocumentModel run(DocumentModel doc) throws ConflictOperationException, IOException {
+    public DocumentModel run(DocumentModel doc) throws ConcurrentUpdateException, IOException {
 
         if (changeToken != null) {
             // Check for dirty update
-            String repoToken = doc.getChangeToken();
-            if (!changeToken.equals(repoToken)) {
-                throw new ConflictOperationException(doc);
-            }
+            save = true;
```

Comment:
Maybe could be stated in the `changeToken` parameter description that it forces the session to be saved, even if the `save` parameter was set to false?